data-collection/data-collection/View Controllers/Rich Popup View Controllers/Rich Popup View Controller/Extensions/AGSPopupAttachmentSize+PreferredSize.swift

```diff
@@ -14,17 +14,17 @@
 
 import ArcGIS
 
-extension AGSPopupAttachmentSize {
-    
-    private static let userDefaultsKey = "AGSPopupAttachmentSize.StaticUserDefaultKey"
-    
-    func storeDefaultPopupAttachmentSize() {
-        UserDefaults.standard.set(self.rawValue, forKey: AGSPopupAttachmentSize.userDefaultsKey)
-    }
-    
-    static func retrieveDefaultPopupAttachmentSize() -> AGSPopupAttachmentSize {
-        
-        let storedRaw = UserDefaults.standard.integer(forKey: AGSPopupAttachmentSize.userDefaultsKey)
-        return AGSPopupAttachmentSize(rawValue: storedRaw) ?? .actual
+extension String {
+    fileprivate static let defaultAttachmentSize = "AGSPopupAttachmentSize.PreferredAttachmentSize.UserDefaultKey"
+}
```

Comment:
Since `private` and `file private` are the same at the top level of a file, it is recommended to just use `private` and use `fileprivate` only when `private` won't cut it. Also, in general, I recommend putting the access control keyword on the extension when possible. This makes it very handy to add additional private members in the future to extensions like this, where all the members are just private implementation details.

```suggestion
private extension String {
    static let defaultAttachmentSize = "AGSPopupAttachmentSize.PreferredAttachmentSize.UserDefaultKey"
}
```